### DISCUSSION TITLE
django.request log Bad Request: /control

### DISCUSSION THREAD

**Discussion by klarkcent** [pretix/pretix#2682] [Q&A] [answered]
After following  https://docs.pretix.eu/en/latest/admin/installation/manual_smallscale.html
I get 
 django.request log Bad Request: /control
for all requests to https://pretix.mysite.info/control
Research on the internet shows that is is related to 
django configuration of ALLOWED_HOSTS  

When I change
proxy_set_header Host $http_host;
to 
proxy_set_header Host localhost;
It works but then I get of cause
WARNING 2022-06-12 16:56:15,394 django.security.csrf log Forbidden (Referer checking failed - no Referer.): /control/login

So where I can set ALLOWED_HOSTS  to
ALLOWED_HOSTS = ["pretix.mysite.info,"localhost"]
?

https://github.com/pretix/pretix/discussions/2601
does not really help

/etc/pretix/pretix.conf
[pretix]
instance_name=Pretix-Ticket-Service
url=https://pretix.mysite.info
....

/etc/nginx/conf.d/pretix-ssl-nginx.conf
...
 location / {
        proxy_pass http://localhost:8345;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
        proxy_set_header X-Forwarded-Proto https;
        proxy_set_header Host $http_host;
    }
...

**Comment by klarkcent** [comment, marked as answer]
it is actually 
etc/pretix/pretix.cnf 
note
etc/pretix/pretix.conf

**Reply by raphaelm** [reply]
actually it should be .cfg 🧐
